types/src/response.rs

```diff
@@ -93,14 +93,14 @@ mod tests {
 
 	#[test]
 	fn serialize_call_response() {
-		let ser = serde_json::to_string(&Response { jsonrpc: TwoPointZero, result: "ok", id: Id::Number(1) }).unwrap();
+		let ser = serde_json::to_string(&Response { jsonrpc: Some(TwoPointZero), result: "ok", id: Id::Number(1) }).unwrap();
 		let exp = r#"{"jsonrpc":"2.0","result":"ok","id":1}"#;
 		assert_eq!(ser, exp);
 	}
 
 	#[test]
 	fn deserialize_call() {
```

Comment:
Can you add a similar test without the `jsonrpc version field`? 

I'm quite certain that it will work but doesn't hurt to have a test for it :)